packages/childchain/contracts/bridge.sol

```diff
@@ -9,16 +9,18 @@ pragma experimental ABIEncoderV2;
  * For more details, check out the docs in /docs/childchain.md
  */
 contract Bridge {
```

Comment:
you could add a function `circulatingSupply()` here, and then calculate `0xff..ff - address(this).balance`